src/main/java/racingcar/domain/Participants.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import racingcar.domain.car.Car;
+import racingcar.domain.car.Move;
+import racingcar.domain.random.RandomGenerator;
+import racingcar.view.UserConsole;
+
+public class Participants {
+
+    private static final String NAME_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
+    private static final String ERROR_LOG = "[ERROR] ";
+    private final List<Car> cars;
+    private final Move move;
+
+    private Participants(List<Car> cars) {
+        this.cars = cars;
+        this.move = Move.get();
+    }
+
+    public static Participants getInstance(List<Car> cars) {
+        return new Participants(cars);
+    }
+
+    public static Participants toParticipants(RacingResult racingResult) {
+        return Participants.getInstance(racingResult.getResult());
+    }
+
+    public static Participants createCars() {
+        String[] names = UserConsole.getConsoleTextFrom(NAME_MESSAGE).split(",");
+        List<Car> cars = new ArrayList<>();
+        try {
+            for (String name : names) {
+                name = name.trim();
+                Car car = Car.from(name);
+                cars.add(car);
+            }
+        } catch (IllegalArgumentException e) {
+            System.out.println(ERROR_LOG + e.getMessage());
+            return createCars();
+        }
+        return Participants.getInstance(cars);
+    }
+
+    public RacingResult race() {
+        int carSize = cars.size();
+        for (int i = 0; i < carSize; i++) {
+            moveCarIfPositionChanged(i, move.isSatisfiedMoveCondition(new RandomGenerator()));
+        }
```

Comment:
index로 직접 cars의 요소에 접근할 필요가 없는 것 같아요! 😃
아래와 같이 변경해보면 어떨까요 ? 🤔

```suggestion
        for (Car car : cars) {
            if (move.isSatisfiedMoveCondition()) {
                car.go();
            }
        }
```

개인적으로는 random 패키지의 `RandomFactory`, `RandomGenerator`, `RandomStrategy`의 클래스(혹은 인터페이스)가 존재하는데, `RandomFactory`는 불필요하다고 생각됩니다. 👀 
오히려 불필요하게 `getRandom` 메서드의 인자로`RandomStrategy`를 전달받는 것이 불필요하다고 느껴져요. 🥲

또한 `MoveGen`과 `NoMoveGen` 클래스의 경우 테스트를 위한 클래스라고 생각되는데, 이런 클래스의 경우 test 패키지의 경로에 두시는게 더 좋지않을까요? 🤔